js/modules/k6/grpc/client_test.go

```diff
@@ -669,11 +689,16 @@ func TestClient(t *testing.T) {
 
 			ts := setup(t)
 
-			// create and attach the GRPC module to goja
 			ctx := common.WithRuntime(context.Background(), ts.rt)
```

Comment:
You should actually be able to remove this and maybe have a cancelable context and cancel it :thinking: